Build broken on s390x

I used to try build the extension for [Alpinelinux](https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/31206) and faced build failure on [s390x arch](https://gitlab.alpinelinux.org/alpine/aports/-/jobs/638248)

```
/builds/alpine/aports/testing/php81-pecl-teds/src/teds-1.1.0/teds_intvector.c: In function 'zim_Teds_IntVector___unserialize':
/builds/alpine/aports/testing/php81-pecl-teds/src/teds-1.1.0/teds_intvector.c:650:20: error: #if with no expression
  650 | #if WORDS_BIGENDIAN
      |                    ^
/builds/alpine/aports/testing/php81-pecl-teds/src/teds-1.1.0/teds_intvector.c: In function 'teds_create_string_from_entries_int16':
/builds/alpine/aports/testing/php81-pecl-teds/src/teds-1.1.0/teds_intvector.c:697:20: error: #if with no expression
  697 | #if WORDS_BIGENDIAN
      |                    ^
/builds/alpine/aports/testing/php81-pecl-teds/src/teds-1.1.0/teds_intvector.c: In function 'teds_create_string_from_entries_int32':
/builds/alpine/aports/testing/php81-pecl-teds/src/teds-1.1.0/teds_intvector.c:714:20: error: #if with no expression
  714 | #if WORDS_BIGENDIAN
      |                    ^
/builds/alpine/aports/testing/php81-pecl-teds/src/teds-1.1.0/teds_intvector.c: In function 'teds_create_string_from_entries_int64':
/builds/alpine/aports/testing/php81-pecl-teds/src/teds-1.1.0/teds_intvector.c:731:20: error: #if with no expression
  731 | #if WORDS_BIGENDIAN
      |                    ^
make: *** [Makefile:246: teds_intvector.lo] Error 1
```

--- a/teds_bitset.c
+++ b/teds_bitset.c
@@ -633,7 +633,7 @@ static zend_always_inline zend_string *teds_create_string_from_entries_int8(cons
 }
 
 static zend_always_inline zend_string *teds_create_string_from_entries_int16(const char *raw, const size_t len) {
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 	zend_string *const result = zend_string_alloc(len * sizeof(int16_t), 0);
 	uint16_t *dst = (uint16_t *)ZSTR_VAL(result);
 	const uint16_t *src = (const uint16_t *)raw;
@@ -650,7 +650,7 @@ static zend_always_inline zend_string *teds_create_string_from_entries_int16(con
 }
 
 static zend_always_inline zend_string *teds_create_string_from_entries_int32(const char *raw, const size_t len) {
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 	zend_string *const result = zend_string_alloc(len * sizeof(int32_t), 0);
 	uint32_t *dst = (uint32_t *)ZSTR_VAL(result);
 	const uint32_t *src = (const uint32_t *)raw;
@@ -667,7 +667,7 @@ static zend_always_inline zend_string *teds_create_string_from_entries_int32(con
 }
 
 static zend_always_inline zend_string *teds_create_string_from_entries_int64(const char *raw, const size_t len) {
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 	zend_string *const result = zend_string_alloc(len * sizeof(int64_t), 0);
 	uint64_t *dst = (uint64_t *)ZSTR_VAL(result);
 	const uint64_t *src = (const uint64_t *)raw;
@@ -1057,7 +1057,7 @@ PHP_METHOD(Teds_BitSet, offsetSet)
 }
 
 #define teds_bitset_convert_8(x) (x)
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 #define teds_bitset_convert_16(x) teds_bswap_16((x))
 #define teds_bitset_convert_32(x) teds_bswap_32((x))
 #define teds_bitset_convert_64(x) teds_bswap_64((x))
@@ -1234,7 +1234,7 @@ static zend_always_inline void teds_bitset_entries_unshift_single(teds_bitset_en
 	ZEND_ASSERT(array->bit_size <= array->bit_capacity);
 	size_t n = old_size >> 3;
 	uint8_t *const entries_bits = array->entries_bits;
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 	for (; n > 0; n--) {
 		entries_bits[n] = (entries_bits[n] << 1) | (entries_bits[n - 1] >> 7);
 	}

--- a/teds_intvector.c
+++ b/teds_intvector.c
@@ -647,7 +647,7 @@ PHP_METHOD(Teds_IntVector, __unserialize)
 			array->entries_int8 = (int8_t *)values;
 			array->size = num_elements;
 			array->capacity = num_elements;
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 			/* TODO: Can probably optimize this with C restrict keyword to indicate memory doesn't overlap? */
 			if (type_tag != TEDS_INTVECTOR_TYPE_INT8) {
 				if (type_tag == TEDS_INTVECTOR_TYPE_INT64) {
@@ -694,7 +694,7 @@ static zend_always_inline zend_string *teds_create_string_from_entries_int8(cons
 }
 
 static zend_always_inline zend_string *teds_create_string_from_entries_int16(const char *raw, const size_t len) {
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 	zend_string *const result = zend_string_alloc(len * sizeof(int16_t), 0);
 	uint16_t *dst = (uint16_t *)ZSTR_VAL(result);
 	const uint16_t *src = (const uint16_t *)raw;
@@ -711,7 +711,7 @@ static zend_always_inline zend_string *teds_create_string_from_entries_int16(con
 }
 
 static zend_always_inline zend_string *teds_create_string_from_entries_int32(const char *raw, const size_t len) {
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 	zend_string *const result = zend_string_alloc(len * sizeof(int32_t), 0);
 	uint32_t *dst = (uint32_t *)ZSTR_VAL(result);
 	const uint32_t *src = (const uint32_t *)raw;
@@ -728,7 +728,7 @@ static zend_always_inline zend_string *teds_create_string_from_entries_int32(con
 }
 
 static zend_always_inline zend_string *teds_create_string_from_entries_int64(const char *raw, const size_t len) {
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 	zend_string *const result = zend_string_alloc(len * sizeof(int64_t), 0);
 	uint64_t *dst = (uint64_t *)ZSTR_VAL(result);
 	const uint64_t *src = (const uint64_t *)raw;

--- a/teds_lowmemoryvector.c
+++ b/teds_lowmemoryvector.c
@@ -822,7 +822,7 @@ PHP_METHOD(Teds_LowMemoryVector, __unserialize)
 			array->entries_int8 = (int8_t *)values;
 			array->size = num_elements;
 			array->capacity = num_elements;
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 			/* TODO: Can probably optimize this with C restrict keyword to indicate memory doesn't overlap? */
 			if (type_tag != LMV_TYPE_INT8) {
 				if (type_tag == LMV_TYPE_INT64 || type_tag == LMV_TYPE_DOUBLE) {
@@ -869,7 +869,7 @@ static zend_always_inline zend_string *teds_create_string_from_entries_int8(cons
 }
 
 static zend_always_inline zend_string *teds_create_string_from_entries_int16(const char *raw, const size_t len) {
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 	zend_string *const result = zend_string_alloc(len * sizeof(int16_t), 0);
 	uint16_t *dst = (uint16_t *)ZSTR_VAL(result);
 	const uint16_t *src = (const uint16_t *)raw;
@@ -886,7 +886,7 @@ static zend_always_inline zend_string *teds_create_string_from_entries_int16(con
 }
 
 static zend_always_inline zend_string *teds_create_string_from_entries_int32(const char *raw, const size_t len) {
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 	zend_string *const result = zend_string_alloc(len * sizeof(int32_t), 0);
 	uint32_t *dst = (uint32_t *)ZSTR_VAL(result);
 	const uint32_t *src = (const uint32_t *)raw;
@@ -903,7 +903,7 @@ static zend_always_inline zend_string *teds_create_string_from_entries_int32(con
 }
 
 static zend_always_inline zend_string *teds_create_string_from_entries_int64(const char *raw, const size_t len) {
-#if WORDS_BIGENDIAN
+#ifdef WORDS_BIGENDIAN
 	zend_string *const result = zend_string_alloc(len * sizeof(int64_t), 0);
 	uint64_t *dst = (uint64_t *)ZSTR_VAL(result);
 	const uint64_t *src = (const uint64_t *)raw;

